Sources/XcodeProj/Project/XCSharedData.swift

```diff
@@ -36,9 +36,9 @@ public final class XCSharedData: Equatable {
         if !path.exists {
             throw XCSharedDataError.notFound(path: path)
         }
-        schemes = path.glob("xcschemes/*.xcscheme")
+        schemes = Self.schemesPath(path).glob("*.xcscheme")
             .compactMap { try? XCScheme(path: $0) }
-        breakpoints = try? XCBreakpointList(path: path + "xcdebugger/Breakpoints_v2.xcbkptlist")
+        breakpoints = try? XCBreakpointList(path: Self.breakPointsPath(path))
```

Comment:
This is now using static path methods so these string literals only have to be defined once